src/main/frontend/handler/events.cljs

```diff
@@ -71,12 +70,13 @@
     close-fn)))
 
 (defmethod handle :graph/added [[_ repo]]
-  ;; add ast/version to db
-  (let [conn (conn/get-conn repo false)
-        ast-version (d/datoms @conn :aevt :ast/version)]
+  ;; TODO: add ast/version to db
```

Comment:
This looked like a todo so I added that. I could also remove the let bindings completely if it's not a TODO